src/css/profile/mobile/common/listview.less

```diff
@@ -173,6 +173,13 @@ tau-expandable {
 				color: var(--text-secondary-color);
 				line-height: 15 * @px_base;
 
+				// line height between title and sub
+				&+.ui-li-text-title::before {
+					content: "";
+					display: block;
+					height: 4 * @px_base;
+				}
+
```

Comment:
You already set the "ui-li-text-sub::before" by "height: 4 * @px_base;"
Is it also needed?

				// line height between title and sub
				&+.ui-li-text-sub::before {
					content: "";
					display: block;
					height: 4 * @px_base;
				}